src/ui/states/state-needs-payment-info.svelte

```diff
@@ -17,12 +17,15 @@
   import IconLock from "../assets/icon-lock.svelte";
   import { Colors } from "../../assets/colors";
   import ProcessingAnimation from "../processing-animation.svelte";
+  import { Product, PurchaseOption } from "../../main";
```

Comment:
I think we should import these from the model classes. Less chances of some circular reference.
```suggestion
  import { Product, PurchaseOption } from "../../entities/offerings";
```